bay_trimesh/anaconda-project.yml

```diff
@@ -0,0 +1,53 @@
+name: bay_trimesh
+description: Visualizing water depth into the Chesapeake and Delaware Bays
+maintainers:
+- philippjfr
+
+channels:
+- pyviz
+
+dependencies: &deps
+- colorcet ==2.0.1
+- datashader ==0.7.0
+- geoviews ==1.6.2
+- holoviews ==1.12.2
+- notebook ==5.7.8
+- pandas ==0.24.2
+
+packages: *deps
+
+platforms:
+- linux-64
+- osx-64
+- win-64
+
+commands:
+  notebook:
+    notebook: bay_trimesh.ipynb
+    env_spec: pyviz-topics-bay-trimesh
```

Comment:
Is the env_spec required here?  I'd like this file to be pared down to the absolute essentials, because otherwise unneeded stuff will just get copied everywhere.

Reply:
right. I just pushed.